versions/1/data_stream/_dev/test/pipeline/config_raw.spec.yml

```diff
@@ -0,0 +1,20 @@
+spec:
```

Comment:
I think the config is applicable in both modes now: raw and JSON. It's described also in docs for pipeline tests.

Reply:
There appears to be a slight difference in the configs for the raw and JSON modes.

Raw: https://github.com/elastic/package-spec/blob/930b44726a527ac599f55eebdfca114f049bdeff/versions/1/data_stream/_dev/test/pipeline/spec.yml#L14-L36

JSON: https://github.com/elastic/package-spec/blob/930b44726a527ac599f55eebdfca114f049bdeff/versions/1/data_stream/_dev/test/pipeline/spec.yml#L37-L55

It looks like the raw mode supports `multiline` whereas the JSON mode does not. 

That's why I had to end up creating two different spec files: `config_raw.spec.yml` and `config_json.spec.yml`.

Reply:
Ah, I see! In this case I think it's correct. Sorry about confusion.